db/pom.xml

```diff
@@ -82,20 +82,20 @@
             <artifactId>postgresql</artifactId>
         </dependency>
 
-        <!-- test logging -->
+        <!-- flyway bundle -->
         <dependency>
-            <groupId>ch.qos.logback</groupId>
-            <artifactId>logback-classic</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>com.opentable.components</groupId>
-            <artifactId>otj-pg-embedded</artifactId>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
         </dependency>
         <dependency>
             <groupId>org.flywaydb</groupId>
             <artifactId>flyway-core</artifactId>
-            <scope>test</scope>
+        </dependency>
```

Comment:
need to version bump based on this as downstream projects may have flyway in test scope and dependency-scope plugin will complain about reduced scope